src/persist/src/indexed/encoding.rs

```diff
@@ -558,18 +563,16 @@ impl BlobUnsealedBatch {
 // interface for this.
 impl Codec for BlobUnsealedBatch {
     fn codec_name() -> String {
-        "bincode[BlobUnsealedBatch]".into()
+        "UnsealedBatch".into()
```

Comment:
probably nit: why is this not sth like `"parquet[UnsealedBatch]"`

Reply:
no reason. switched to that